config/cars/kunos/ks_nissan_370z.ini

```diff
@@ -43,7 +43,7 @@ Refraction=1
 
 [SHADER_REPLACEMENT_...]
 MATERIALS = EXT_lights_chrome
-LAYER = 5
+;LAYER = 10 ; i don't know what the point of this line was, i could not find any visual difference, and it only caused the chrome to be missing in some cases.
```

Comment:
i really dont know what this is meant to do, from my testing it just seemed to cause the metal to dissapear in some conditions, even at different values.

Reply:
This parameter ties into ksLayer, affected by your World Detail settings iirc. The maximum is Layer 3, though I believe this was pushed sometime ago to prevent the metal from disappearing under certain angles.